sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -55,6 +55,22 @@ bcs.registerStructType('TransferObjectTx', {
   object_ref: 'SuiObjectRef',
 });
 
+/**
+ * Transaction type used for transferring Sui.
+ */
+export type TransferSuiTx = {
+  TransferSui: {
+    recipient: string;
+    amount: number | null;
+  };
+};
+
+bcs.registerStructType('TransferSuiTx', {
+  recipient: 'SuiAddress',
+  // TODO: figure out how to represent Option<u64> field
```

Comment:
@damirka  do you know how to do this?

Reply:
Yup. Option is an enum. I would suggest something like:
```
bcs.registerEnumType('Option', {
  none: null,
  some: 'u64'
});
```